app/config/parameters.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$container->setParameter('database_host', getenv('APP__DATABASE_HOST'));
```

Comment:
maybe ``CONFERIA_DATABASE_HOST``? If this file is going to be a part of repo then env variables could be a bit more unique.